ocs_ci/ocs/acm/acm.py

```diff
@@ -41,19 +44,35 @@ def import_cluster_ui(self, cluster_name, kubeconfig_location):
 
         """
         self.navigate_clusters_page()
-        self.do_click(self.page_nav["Import_cluster"])
+        time.sleep(40)
```

Comment:
We should use explict wait functions instead of hard coded sleep

Reply:
@am-agrawa ,  There were runs where element found returns true but successive ops fails, hence I took this blind sleep approach.

Reply:
@shylesh what if as part of such an explicit wait function, we will add a sample operation that would give another assurance, on top of finding the element?

Reply:
@ebenahar , that could be done, earlier in some other places I had added few more checks like checking the URL along with the element itself. I ll check what else could be added here

Reply:
@ebenahar , I have replaced the sleep with an implicit url wait , hope this is foolproof.